Mathlib/Probability/Density.lean

```diff
@@ -63,30 +63,60 @@ namespace MeasureTheory
 variable {Ω E : Type*} [MeasurableSpace E]
 
 /-- A random variable `X : Ω → E` is said to `HasPDF` with respect to the measure `ℙ` on `Ω` and
-`μ` on `E` if there exists a measurable function `f` such that the push-forward measure of `ℙ`
-along `X` equals `μ.withDensity f`. -/
+`μ` on `E` if the push-forward measure of `ℙ` along `X` is absolutely continuous with respect to
+`μ` and they `HaveLebesgueDecomposition`. -/
 class HasPDF {m : MeasurableSpace Ω} (X : Ω → E) (ℙ : Measure Ω)
     (μ : Measure E := by volume_tac) : Prop where
-  -- porting note: TODO: split into fields `Measurable` and `exists_pdf`
-  pdf' : Measurable X ∧ ∃ f : E → ℝ≥0∞, Measurable f ∧ map X ℙ = μ.withDensity f
+  pdf' : Measurable X ∧ HaveLebesgueDecomposition (map X ℙ) μ ∧ map X ℙ ≪ μ
```

Comment:
It was decided that even in probability theory, we should stick with ae measurable instead of measurable, so can you change the definition to require ae measurable instead?